clearwater-cassandra/usr/share/clearwater/clearwater-cluster-manager/plugins/cassandra_plugin.py

```diff
@@ -75,6 +75,16 @@ def join_cassandra_cluster(cluster_view,
         doc["listen_address"] = ip
         doc["seed_provider"][0]["parameters"][0]["seeds"] = seeds_list_str
         doc["endpoint_snitch"] = "GossipingPropertyFileSnitch"
+       
+        # Work out the timeout from the target_latency_us value (assuming
+        # 100000 if it isn't set)
+        get_latency_cmd = "target_latency_us=100000; . /etc/clearwater/config; echo -n $target_latency_us"
+        latency = subprocess.check_output(get_latency_cmd,
+                                          shell=True,
+                                          stderr=subprocess.STDOUT)
+
+        timeout = int(latency) * 4 / 5
```

Comment:
`latency` is effectively an external input, so you need to validate it better in case its not an integer. 

Python uses the "its easier to ask for forgiveness than permission" approach to error handling. So you should assume its going to be an integer and catch and handle the `ValueError` that is raised if it isn't. (btw, C tends to use the "look before you leap" aproach, where you validate everything up front).